src/config.js

```diff
@@ -14,7 +14,9 @@ export const SUPPORTED_BROWSERS = {
 };
 
 export const CLIENT_ID_ALIAS = {
-    sb: 'AZDxjDScFpQtjWTOUtWKbyN_bDt4OgqaF4eYXlewfBP4-8aqX3PiV8e1GWU6liB2CUXlkA59kJXE7M6R'
+    sandbox:    'AZDxjDScFpQtjWTOUtWKbyN_bDt4OgqaF4eYXlewfBP4-8aqX3PiV8e1GWU6liB2CUXlkA59kJXE7M6R',
```

Comment:
Could you create a variable for this client-id and reuse it for sb, sandbox, and test? 

I'm thinking about bundle size. I don't think babel would be smart enough to transpile this down to one string variable in the current format.

Reply:
good catch. done! 👍